pyproject.toml

```
@@ -62,7 +62,9 @@ extend-select = [
     "D402",
     "D403",
     "D412",
-    "D419"
+    "D419",
+    "TCH001", # typing-only-first-party-import
+    "TCH002", # typing-only-third-party-import
```

```suggestion
    "TCH001",  # typing-only-first-party-import
    "TCH002",  # typing-only-third-party-import
```

Not actually revelant but this matches how Black formats Python code